gdal_subsetter/transform.py

```diff
@@ -1731,17 +1750,17 @@ def _process_geogcs(ds_in, dst):
         # open infile
         ds_in = gdal.Open(infile)
         # create a output nc file
-        dst = Dataset(outfile, mode='w', format="NETCDF4")
+        dst = Dataset(outfile, mode='w', format='NETCDF4')
         # define global attributes
         dst.title = ''
         dst.institution = 'Alaska Satellite Facility'
         dst.source = ''
         dst.references = ''
         dst.comment = ''
-        dst.history = "{datetime} (UTC)".format(datetime=datetime.utcnow().strftime("%d/%m/%Y %H:%M:%S"))
-        dst.GDAL = "Version {ver}".format(ver=gdal.__version__)
+        dst.history = f'{datetime.utcnow():%d/%m/%Y %H:%M:%S} (UTC)'
```

Comment:
I thought this was a pretty cool trick - I didn't previously know that f-strings were smart enough to format datetimes, but I suppose they should handle them as datetimes are part of the Python standard library.